app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap

```diff
@@ -6,7 +6,7 @@ exports[`Metadata should not render metadata without value 1`] = `
 <React.Fragment>
   <dl
     className="metadata-type-default "
-    key="Label"
+    key="undefined_0"
```

Comment:
Also, every index is _0.  Am I missing somewhere where it is tested that further indexes would receive a subsequent number?